### PR TITLE
Performance tests for CF with MySQL database

### PR DESCRIPTION
**Pull request by jochenehret**

### CODE REVIEW

File: concourse/deploy-cf-perftest.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ resources:
     type: git
     source:
       uri: https://github.com/cloudfoundry/cf-performance-tests.git
-      branch: main
+      branch: CFP-1806
```
Comment (johha) [inline concourse/deploy-cf-perftest.yml:53]: branch should be `main`

File: concourse/deploy-cf-perftest.yml [hunk 2]
```diff
@@ -98,7 +98,7 @@ jobs:
             source:
               repository: cloudfoundry/cf-deployment-concourse-tasks
             version:
-              digest: sha256:90349a43f1a254d9e458a519fe2d3acc5c655d90ac82a418d009c923df0bcc26
+              digest: sha256:e5a5877fa88caa76b3f8dafe7d15111b295a97506780dfe16cbf1e05a3e4fbad
```
Comment (johha) [inline concourse/deploy-cf-perftest.yml:101]: Do we need a job to update those digests regularly?

**Review by johha — APPROVED** [review]

File: concourse/deploy-cf-perftest.yml (hunk 2 above)
Comment (jochenehret) [reply concourse/deploy-cf-perftest.yml:101]: Possibly yes. The next versions will however come with Ginkgo v2, so we should first migrate our tests.

**Comment by will-gant** [comment]
I've pushed a fair few changes to this just now, mostly squashed into one big commit. The big high-level changes are:

* We now deploy (almost) all the infrastructure we need in each run of the pipeline. This includes DNS and a (self-signed) TLS certificate.
* Any 'shared' infra that _must_ be manually created as a one-off is now done so via CloudFormation that can be run with a bash script in `/scripts`
* At the end of each test run we tear down the Cloud Foundry foundation, the director and all infrastructure created in the pipeline
* Tests for mysql and postgres are run serially in the same pipeline run, rather than in separate pipelines, with a `bosh deploy` in between
* All test environments now use S3 for the Cloud Controller blobstore
* Test results and charts all now live in `/results` in a hierarchy based on Cloud Controller type (go/rails) and CCDB type (postres/mysql).
* Charts are generated and pushed to git by a separate `performance-tests-charts` pipeline. This avoids the conflicts that occur if one pipeline pulls the repo, spends an hour running its tests, and then tries to update the same .png file that some other pipeline has changed in the meantime.

File: concourse/deploy-cf-perftest.yml (hunk 2 above)
Comment (will-gant) [reply concourse/deploy-cf-perftest.yml]: Do we have a particular reason for not expecting that the latest cf-deployment release will be deployable with the latest release of cf-deployment-concourse-tasks running on its latest image?

I'm thinking we might as well just let the pipeline reveal any incompatibilities to us and just fix them as and when they occur. Alternatively, if an actual compatibility matrix exists somewhere (_and we could write something to generate one in the cf-deployment pipeline..._) we could use that instead.

**Review by jochenehret — COMMENTED** [review]
So far I've only found a few issues in the documentation.

**Comment by will-gant** [comment]
@jochenehret After that messy rebase with my latest changes and test results generated during development this is now ready for you. 

Had to change the author on those to myself because of the requirement for committers to have signed a CLA.

A couple of small things:
* A security improvement I'd suggest is using deploy keys for this repo (and also cloudfoundry/cf-performance-tests) so we can use them in the pipeline. Just minimises the scope for damage should someone get access to them.
* Is there any reason not to publicly-expose these pipelines (`fly expose-pipeline`) and [also their build output](https://concourse-ci.org/jobs.html#schema.job.public)? That output doesn't include anything sensitive, and I think this practice is pretty common for other CFF pipelines.

File: README.md [hunk 3]
```diff
@@ -1,95 +1,63 @@
...
 2. Detailed with most recent runs. Same as 1, but only contains the last 15 runs.
 3. Simplified: Chart only contains the average cf api execution time.
 4. Simplified with most recent runs: same as 3, only contains the last 15 runs.
 
 ## General information
-The AWS account and domain used to host the BBL and CF foundation is currently owned by SAP. It might move to a community owned account in the future. A description of how this was set up can be found [here](docs/manual-setup.md).
+The AWS account and domain used to host the BBL and CF foundation is controlled by SAP, but may move to a community owned account in the future. See [here](docs/manual-setup.md) for information on the manual steps that were followed to get the tests automated.
 
-## Automatic Setup / Destruction
+The pipeline currently runs on [bosh.ci.cloudfoundry.org](https://bosh.ci.cloudfoundry.org/), but it is planned to migrate this in the near future to a Concourse controlled by the Cloud Foundry Foundation's [App Runtime Deployments Working Group](https://github.com/cloudfoundry/community/blob/main/toc/working-groups/app-runtime-deployments.md).
```
[the remark this answers is not in the document]
Comment (will-gant) [reply README.md:41]: Cool changing this

File: README.md [hunk 4]
```diff
@@ -1,95 +1,63 @@
...
-  -c ./concourse/destroy-cf-perftest.yml
-```
-
-
-The deploy pipeline runs `bbl up` followed by a `bosh deploy` for the CF deployment. Then it executes the performance tests and generates visual charts. Test results and charts are automatically uploaded to github. The pipeline also runs the CF Acceptance Tests and finally destroys the "cf" BOSH deployment to save cost.
-
-The destroy pipeline first deletes all BOSH deployments and then runs `bbl destroy` to delete all IaaS resources. Use this only if you want to tear down the complete environment.
-
-
-## Troubleshooting
```
[the remark this answers is not in the document]
Comment (will-gant) [reply README.md:79]: Alright adding a troubleshooting section back in

**Pull request merged**